edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Amqp.Test/TwinReceivingLinkHandlerTest.cs

```diff
@@ -86,7 +86,7 @@ public async Task ProcessDeleteOperationMessageTest()
             var messageConverter = Mock.Of<IMessageConverter<AmqpMessage>>();
             var identity = Mock.Of<IIdentity>(i => i.Id == "d1");
             var metadataStore = Mock.Of<IMetadataStore>();
-            var twinReceivingLinkHandler = new TwinReceivingLinkHandler(identity, receivingLink, requestUri, boundVariables, connectionHandler, messageConverter, metadataStore);
+            var twinReceivingLinkHandler = new TwinReceivingLinkHandler(identity, receivingLink, requestUri, boundVariables, connectionHandler, messageConverter, metadataStore, false);
```

Comment:
Do we need to expand these tests to cover both enabled and disabled?

Reply:
it just passes through a setting to a library (the amqp library) to control a behavior that is invisible for us, I don't think it needs testing. I am not sure if we could test other than that the parameter flows down.